ffi/firewood.go

```diff
@@ -1,179 +1,186 @@
+// Package firewood provides a Go wrapper around the [Firewood] database.
+//
+// [Firewood]: https://github.com/ava-labs/firewood
 package firewood
 
-// #cgo LDFLAGS: -L../target/release -L/usr/local/lib -lfirewood_ffi -lm
+// #cgo LDFLAGS: -L${SRCDIR}/../target/release -L/usr/local/lib -lfirewood_ffi -lm
```

Comment:
Please add a comment as to why we need -lm here to deter someone else from thinking about removing it. It's required on linux.

Reply:
Done